x-pack/plugins/security_solution/public/detections/hooks/trigger_actions_alert_table/use_bulk_actions.tsx

```diff
@@ -88,5 +89,11 @@ export const getBulkActionHook =
       refetch: refetchGlobalQuery,
     });
 
-    return [...alertActions, timelineAction];
+    const { alertTagsItems, alertTagsPanels } = useBulkAlertTagsItems({
```

Comment:
I know this file does not have any unit tests to add to, but it really should have them on the new and existing behavior